Cargo.toml

```diff
@@ -53,7 +53,7 @@ percent-encoding = "2.0"
 remove_dir_all = "0.5.2"
 rustfix = "0.5.0"
 same-file = "1"
-semver = { version = "0.9.0", features = ["serde"] }
+semver = { git = "https://github.com/illicitonion/semver.git", rev = "f1f912703f67ed63b751c525839731a90239bdcf", features = ["serde"] }
```

Comment:
I'd probably want an upstream release of this (including https://github.com/steveklabnik/semver/pull/205) before merging this PR, but figured I'd see if there was appetite for it first.